mappings/net/minecraft/client/render/block/entity/HangingSignBlockEntityRenderer.mapping

```diff
@@ -0,0 +1,25 @@
+CLASS net/minecraft/unmapped/C_zcgkklbo net/minecraft/client/render/block/entity/HangingSignBlockEntityRenderer
+	FIELD f_gzejleyy PLANK Ljava/lang/String;
+	FIELD f_kydnbzbo CHAIN_R2 Ljava/lang/String;
```

Comment:
How about `RIGHT_CHAIN(_2)`?

Reply:
its the string name for the model part. I just copied them